chap-cheri-riscv.tex

```diff
@@ -1560,3 +1503,16 @@ \subsection{Separate Capability Register File}
 In practice, only variants of CHERI-RISC-V using a ``merged'' register
 file were implemented in emulators, soft cores, toolchains, and
 operating systems.
+
+\subsection{\DDC{} and \PCC{} Relocation}
+
+CHERI-RISC-V originally specified that legacy memory accesses using
+integer pointers were relocated by \DDC{} and \PCC{} in addition to
+being constrained.  In this model, integer pointers were treated as
+offsets relative to the base of \DDC{} and \PCC{} rather than
+addresses.
+
+The current version of CHERI-RISC-V does not relocate integer
+pointers.  However, it may be desirable to provide optional relocation
+in the future, in particular to permit compartmentalization of hybrid
+code in user mode.
```

Comment:
I wonder if it’s worth spelling this out a bit more -- e.g., explaining why this would work for hybrid code?

Otherwise I think this all looks good!

Reply:
I've added a sentence to say what the relocation would be, not quite sure if that's what you are looking for?  I'm somewhat hesitant to speculate what implementation form relocation might take in the future though could describe that if desired?